docs/modules/nexus/pages/administration.adoc

```diff
@@ -0,0 +1,45 @@
+= Nexus Administration
+
+The `ods-core` repository contains a build recipe for a central Nexus instance. The remainder of this document describes how that setup is done, and how it can be modified and updated.
+
+== Updating Nexus to a new version
+
+The Nexus deployment is using an image built in the central ODS namespace. This image is a customisation of the https://hub.docker.com/r/sonatype/nexus3[official Nexus image on DockerHub].
```

Comment:
Where can the reader learn more about the customization mentioned?

Reply:
Addressed in 2c6cb49.